libs/poco/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2007-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=poco
+PKG_VERSION:=1.6.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/pocoproject/poco.git
+PKG_SOURCE_VERSION:=1cbaf38bbc84159b2ffa87ecc5d8be15d943dc31
```

Comment:
Looks like this commit is on 1.7 branch:  https://github.com/pocoproject/poco/commit/1cbaf3

If it works for you, I would suggest we just use the release tarball from pocoproject.org.  Release versions are supposed to be more stable and maintained.  Making tarballs from git repo requires cloning the whole repo every time.